### PR TITLE
✨(project) add script to execute management commands

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

Execution of django management commands in development environment require to execute multiple commands. And, as developers, by definition, we are lazy.

## Proposal

Add a `bin/manage` script allows to execute directly django management commands inside a docker container.

Usage example : `bin/manage createsuperuser`

### CODE REVIEW

**Comment by lunika** [comment]
You should maybe take a look at what we did in richie, all scripts are based on this [file](https://github.com/openfun/richie/blob/9dde77c6193650b8987c5d33b804ad0dac2cf0c1/bin/_config.sh) and allow to create multiple small scripts dedicated to `run` or `exec` a command and a container. You can then create one more for management command.
WDYT ?

**Comment by madmatah** [comment]
> You should maybe take a look at what we did in richie, all scripts are based on this [file](https://github.com/openfun/richie/blob/9dde77c6193650b8987c5d33b804ad0dac2cf0c1/bin/_config.sh) and allows to create multiple small scripts dedicated to `run` or `exec` a command and a container. You can then create one more for management command.
> WDYT ?

Thanks, I added a commit to follow your suggestions

File: bin/ashley [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-DOCKER_USER="$(id -u):$(id -g)"
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
 
-DOCKER_USER=${DOCKER_USER} docker-compose up -d postgresql
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm dockerize dockerize -wait tcp://postgresql:5432 -timeout 60s
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm --service-ports ashley
+_docker_compose up -d postgresql
+_dc_run dockerize dockerize -wait tcp://postgresql:5432 -timeout 60s
+_dc_run --service-ports ashley
```
Comment (jmaupetit) [inline bin/ashley]: Since we define port mapping in the `docker-compose` configuration, is this option required?

File: env.d/development/common [hunk 2]
```diff
@@ -5,4 +5,5 @@ DJANGO_SECRET_KEY=ThisIsAnExampleKeyForDevPurposeOnly
 
 # Python
 PYTHONUNBUFFERED=1
```
Comment (jmaupetit) [inline env.d/development/common]: Out of scope here, but should be in the Dockerfile instead.

File: bin/superuser [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+_django_manage createsuperuser "$@"
```
Comment (jmaupetit) [inline bin/superuser]: Is this shortcut really required and not too much specific as the manage script almost handles it?

File: bin/ashley (hunk 1 above)
Comment (madmatah) [reply bin/ashley]: Yep, apparently with `docker-compose run`, this option it is required.

`docker-compose status` (with the option):
```
         Name                        Command               State            Ports         
------------------------------------------------------------------------------------------
ashley_ashley_run_1       /usr/local/bin/entrypoint  ...   Up      0.0.0.0:8090->8000/tcp 
ashley_postgresql_1       docker-entrypoint.sh postgres    Up      0.0.0.0:15432->5432/tcp
```

`docker-compose status` (without the option) :
```
         Name                        Command               State            Ports         
------------------------------------------------------------------------------------------
ashley_ashley_run_1       /usr/local/bin/entrypoint  ...   Up                             
ashley_postgresql_1       docker-entrypoint.sh postgres    Up      0.0.0.0:15432->5432/tcp
```

File: env.d/development/common (hunk 2 above)
Comment (madmatah) [reply env.d/development/common]: Ok thanks, I moved it in the `development` image of the Dockerfile.

File: bin/superuser (hunk 3 above)
Comment (madmatah) [reply bin/superuser]: This shortcut already saved precious seconds of my life :smile: :trollface:

File: env.d/development/common (hunk 2 above)
Comment (jmaupetit) [reply env.d/development/common]: It should also stand for production to ensure unbuffering of stderr/stdout, see: https://docs.python.org/3/using/cmdline.html#cmdoption-u

Comment (madmatah) [reply env.d/development/common]: done, thanks ! :+1:

File: bin/ashley (hunk 1 above)
Comment (jmaupetit) [reply bin/ashley]: Oh I see. The alternative would be to use:

```suggestion
_docker_compose up -d ashley
```

Comment (jmaupetit) [reply bin/ashley]: But it's fine as is.

File: bin/superuser (hunk 3 above)
Comment (jmaupetit) [reply bin/superuser]: You lazy boy :trollface:

**Pull request merged**